README.md

```diff
@@ -21,7 +15,7 @@ flutter pub add webp --dev
 
 Add to your pubspec.yaml
 
-#### Default usage
+### Default usage
```

Comment:
Add 2nd level header for "Usage" before this heading. Usage is not part of installing

Reply:
There already is `##Package usage` above